### ISSUE TITLE
How to use pre-trained model and where is post-processing codes?

### ISSUE THREAD

**Issue by landkwon94** [IrohXu/lanenet-lane-detection-pytorch#5]
Hello, Sir. First of all, many thanks for your nice codes!

Because there were so many errors while using original-LaneNet code, I was looking for other version of same network.
And I found your codes, and I saw it works well!

By the way, I have some questions.

1) I want to use pre-trained model (such as trained LaneNet model which was trained with tuSimple datasets), may I ask you direction how to use it in your codes?

2) I was to use post processing codes. Because I want to count the number of lanes from the image. But I cannot find this code... Where can I revise this codes?

3) Also, I want to classify multi-class lane detection.. Is there any config file in your codes? And do I have to revise some parts of the code for multi-class detection?

Thank you again for your codes! I will wait for your reply :)

**Comment by IrohXu** [comment]
Hi, landkwon94:

1. You can use my pre-trained model here: https://github.com/IrohXu/lanenet-lane-detection-pytorch/blob/main/log/best_model.pth 
The model is ENet version and trained with Tusimple dataset. For using the other model trained by yourself,  please change the `--model` of `test.py`.

2. You can add your code inside the `test.py`.

3. Tusimple will not helpful for multi-class lane detection & classification, as it do not label the type of the lane marking. My group is researching for this topic these days and we will submit a new dataset and new transformer-based lane detection model to ECCV.

**Comment by landkwon94** [comment]
Thank you so much for your kind reply! I will carefully do that, and I will wait for your new paper! Thanks again :)
